packages/ove-asset-manager/src/OVE.Service.AssetManager/Domain/IAssetFileOperations.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using OVE.Service.AssetManager.Models;
+
+namespace OVE.Service.AssetManager.Domain {
+    public interface IAssetFileOperations {
+        string ResolveFileUrl(OVEAssetModel asset);
+        // ReSharper disable twice UnusedParameter.Global

```

Comment:
R# is picking up that none of the implemenations of the interface use the two parameters. 
This is because the implemention of this method is marked as NotImplemented

Reply:
This is because moving files between s3 buckets is not recommended and is difficult to implement (e.g. amazon s3 does not provide API or SDK for it) and I'm uncertain it is something we want to support.

Reply:
Ok, it is strange that R# complains at the Interface and not at S3AssetFileOperations. Or probably it understands what `#pragma warning disable 1998` means.

Reply:
R# understands the NotImplementedException and so supresses the instance level message